Automattic/wp-calypso
# Site setup: update image and text

Fixes https://github.com/Automattic/dotcom-forge/issues/8872

## Proposed Changes

* Updates image and text used in design choice step in site setup flow

## Before
<img width="1251" alt="Screenshot 2024-09-03 at 11 19 19" src="https://github.com/user-attachments/assets/61e3657a-6915-4b00-915b-27a3ba04563b">

## After
<img width="1249" alt="Screenshot 2024-09-03 at 11 37 36" src="https://github.com/user-attachments/assets/75ee4e84-a844-47f2-b083-83043e675f4a">

## Why are these changes being made?


* Logical flows project

## Testing Instructions



* Create a test site
* Choose `Promote myself or business` site goal
<img width="1112" alt="Screenshot 2024-09-03 at 11 40 17" src="https://github.com/user-attachments/assets/fc54cd01-eefc-4a9d-adde-0b82c05562eb">
* Confirm new image and text look ok


## Pre-merge Checklist
